Parsers/ASimAuthentication/Parsers/ASimAuthenticationGithub.yaml

```diff
@@ -0,0 +1,57 @@
+Parser:
+  Title: ASIM Authentication parser for <product name>
+  Version: '0.1'
+  LastUpdated: <Mar 03, 2024>
+Product:
+  Name: Github
+Normalization:
+  Schema: Authentication
+  Version: '<current schema version>'
+References:
+- Title: ASIM Authentication Schema
+  Link: https://aka.ms/ASimAuthenticationDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+Description: |
+  This ASIM parser supports normalizing the <product name> logs to the ASIM Authentication normalized schema.
+ParserName: ASim_Authentication_Github
+EquivalentBuiltInParser: _ASim_Authentication_Github
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let parser = (
+    disabled:bool = false
+  )
+  {
+    let EventResultDetailslookup=datatable(EventOriginalResultDetails:string,EventResult:string,EventType:string )
```

Comment:
does this list include all Authentication 'action'? did you validate from github documentation as well?

Reply:
Added few more user actions based on authentication events from https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/security-log-events

Reply:
Updated list from product documentation